app/controllers/posts_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /posts/1 or /posts/1.json
   def show
+    return nil
```

Comment:
why are we returning nil here?

Reply:
Was trying to figure out how this function was working! Forgot to remove the line in the PR